create.go

```diff
@@ -115,6 +126,42 @@ func getConfigs(bundlePath, containerID, console string) (vc.PodConfig, ShimConf
 	return *podConfig, shimConfig, nil
 }
 
+func createCgroupsFiles(cgroupsPath string, pid int) error {
```

Comment:
This function really needs a test. Something like https://github.com/01org/cc-oci-runtime/blob/master/tests/oci_test.c#L1144..L1198